src/formatDate.js

```diff
@@ -8,7 +8,32 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateParts = date.split(fromFormat[3]);
+  const dateObj = {};
+
+  for (let i = 0; i < 3; i++) {
```

Comment:
It's a good practice to avoid magic numbers like '3' in the loop condition. You can use `fromFormat.length - 1` instead, which is more self-explanatory and less error-prone if the format changes in the future.